backend/database_handler/transactions_processor.py

```diff
@@ -158,12 +158,16 @@ def insert_transaction(
 
         self.session.flush()  # So that `created_at` gets set
 
-        transaction_audit_record = TransactionsAudit(
-            transaction_hash=new_transaction.hash,
-            data=self._parse_transaction_data(new_transaction),
+        rollup_transaction_record = RollupTransactions(
+            from_=from_address,
+            to_=to_address,
+            gas=0,
+            gas_price=0,
+            value=value,
+            input=self._transaction_data_to_str(self._parse_transaction_data(new_transaction)),
```

Comment:
@kstroobants here we can store the exact same encoded call data that we receive from the frontend

Reply:
@cristiam86 Do you expect a fix? new_transaction.data contains the call_data. Hence, rollup_transaction also contains it.